javascript/ql/src/semmle/javascript/security/dataflow/ZipSlip.qll

```diff
@@ -56,7 +59,10 @@ module ZipSlip {
     )
   }
 
-  /** A zip archive entry path access, as a source for unsafe zip extraction. */
+  /** Gets a property that is used to get the filename part of an archive entry. */
```

Comment:
I assume it's "path" for `unzip` and "name" for `tar-stream`? This may be worth clarifying in the comment (but of course it's fine to allow both for both packages as you do here.)

Reply:
Good point, done.